schema/entity-record.json

```diff
@@ -54,7 +54,90 @@
           "description": "This may be used to provide a local name for this type of entity, or any further information to identify the type of entity. For example, in Finland 'ministeriö' for a government department."
         }
       },
-      "propertyOrder": 4
+      "propertyOrder": 4,
+      "allOf":[
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "arrangement"
```

Comment:
I'm slightly nervous about defining 'type' as an enum earlier and then using 'const' in these schemas for the logic. It works for immediate purposes, but I can imagine that it might cause problems with some json schema tools. Amy would know better then I.

To be on the safe side we can simply use "enum": ["singleValue"].

Reply:
done!